src/main/kotlin/api/model/Mapper.kt

```diff
@@ -0,0 +1,11 @@
+package api.model
+import api.Lang
+import data.Language
+import persistence.model.LanguageEntity
+
+class Mapper {
+    //maps the local lang data class to the common language data class
+    fun mapToCommon(lang: Lang): Language{
```

Comment:
Might want to change this, mapToCommon is ambiguous, perhaps something like mapToLanguage